### PR TITLE
Epic47

### PR DESCRIPTION
**Pull request by shaliniib**
Datasheet to List/set/array  of objects, 
Datasheet column to  list/set/map

### CODE REVIEW

File: simplity/kernel/src/main/java/org/simplity/kernel/data/MultiRowsSheet.java [hunk 1]
```diff
@@ -626,9 +628,246 @@ public void setWidths(int[] widths) {
...
+		List<Value[]> rowList = getAllRows();
+		for(Value[] row:rowList){
+			Object obj;
+			try {
+				obj = Class.forName(className).newInstance();		
+				java.lang.reflect.Field[] fields = obj.getClass().getDeclaredFields();
+				int i=0;
+				for(java.lang.reflect.Field field:fields){	
+					ReflectUtil.setAttribute(obj, field.getName(), row[i].toString(), false);	
+					i++;
```
Comment (archetana) [inline simplity/kernel/src/main/java/org/simplity/kernel/data/MultiRowsSheet.java]: Verify if the column name is matching the field before setting the value

File: examples/helloworld/src/main/java/org/simplity/helloworld/SheetManipulations.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import java.util.Set;
+
+import org.simplity.helloworld.entity.Orders;
+import org.simplity.kernel.data.DataSheet;
+import org.simplity.kernel.data.MultiRowsSheet;
+import org.simplity.kernel.value.Value;
+import org.simplity.service.ServiceContext;
+import org.simplity.tp.LogicInterface;
+
+public class SheetManipulations implements LogicInterface{
```
Comment (archetana) [inline examples/helloworld/src/main/java/org/simplity/helloworld/SheetManipulations.java]: Improve the SheetManipulations.java example in the helloworld

File: simplity/kernel/src/main/java/org/simplity/kernel/data/MultiRowsSheet.java [hunk 3]
```diff
@@ -626,9 +628,246 @@ public void setWidths(int[] widths) {
...
+		Set<Object> entitySet = new HashSet<Object>();
+		List<Value[]> rowList = this.getAllRows();
+		for(Value[] row:rowList){
+			Object obj;
+			try {
+				obj = Class.forName(className).newInstance();		
+				java.lang.reflect.Field[] fields = obj.getClass().getDeclaredFields();
+				int i=0;
+				for(java.lang.reflect.Field field:fields){	
+					ReflectUtil.setAttribute(obj, field.getName(), row[i].toString(), false);
```
Comment (archetana) [inline simplity/kernel/src/main/java/org/simplity/kernel/data/MultiRowsSheet.java]: Verify if the column name is matching the field before setting the value

**Comment by shaliniib** [comment]
fixed all the issues.

**Pull request closed without merging**